packages/ui/src/forms/Select.tsx

```diff
@@ -24,6 +24,7 @@ interface SelectProps extends ChakraProps {
   tooltip?: string;
   required?: Required;
   isDisabled?: boolean;
+  _placeholder?: string;
```

Comment:
why `_placeholder` while `placeholder` already exists in the default props for `ChakraSelect` ?

Reply:
Some typescript thingie

Reply:
change the prop in the component to be `placeholder`. it should match